### PR TITLE
Add user table

### PR DESCRIPTION
**Pull request by casterweb**
image

### CODE REVIEW

File: src/app/models/hr-profile/services/user-table.service.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { UsersList, UserTable } from 'src/app/interfaces/test';
+import { UserTableUrl } from 'src/constants/route-constant';
+
+interface GetUserTable {
+  results: UserTable[];
+}
+
```
Comment (shuliakleonid) [inline src/app/models/hr-profile/services/user-table.service.ts:11]: I think it must be in another folder

File: src/constants/route-constant.ts [hunk 2]
```diff
@@ -17,11 +17,11 @@ export enum SidebarMenu {
   check = 'Check',
   editor = 'Editor',
   manage = 'Manage',
-  statistics = 'Statistics',
 }
 
 export const BASE_API_URL = 'http://elevel-001-site1.btempurl.com';
 
+export const UserTableUrl = `${BASE_API_URL}/api/User`;
 export const LoginApiUrl = `${BASE_API_URL}/api/User/token`;
```
Comment (shuliakleonid) [inline src/constants/route-constant.ts:25]: delete /api we add this route in BASE_API_URL

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: src/app/models/hr-profile/services/user-table-store.service.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  public usersResults$ = this.usersSubject$.asObservable();
+
+  private set userTableResults(usersResults: UsersList) {
+    this.usersSubject$.next(usersResults);
+  }
+
+  constructor(
+    private userTableService: UserTableService,
+    private authStoreService: AuthStoreService,
```
Comment (ExxZzyy) [inline src/app/models/hr-profile/services/user-table-store.service.ts:22]: I think, you don't need  authStoreService

File: src/constants/route-constant.ts (hunk 2 above)
Comment (casterweb) [reply src/constants/route-constant.ts:25]: image

File: src/app/models/hr-profile/services/user-table.service.ts (hunk 1 above)
Comment (casterweb) [reply src/app/models/hr-profile/services/user-table.service.ts:11]: removed this interface

File: src/app/models/hr-profile/services/user-table-store.service.ts (hunk 3 above)
Comment (casterweb) [reply src/app/models/hr-profile/services/user-table-store.service.ts:22]: removed this authStoreService

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**